modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM720GetAllEndPointsTestCase.java

```diff
@@ -88,80 +88,62 @@ public APIM720GetAllEndPointsTestCase(TestUserMode userMode) {
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
 
-        String fileFormat = ".war";
         super.init(userMode);
         log.info("Test Starting user mode:" + userMode);
 
-        String storeURLHttp = storeUrls.getWebAppURLHttp();
-        apiStore = new APIStoreRestClient(storeURLHttp);
-
-        String publisherURLHttp = publisherUrls.getWebAppURLHttp();
-        apiPublisher = new APIPublisherRestClient(publisherURLHttp);
+        //publisher login
+        apiPublisher = new RestAPIPublisherImpl(publisherContext.getContextTenant().getContextUser().getUserName(),
```

Comment:
We should be able to remove these lines as these are initialized in the base class.

Reply:
Removed